charts/vector/templates/_pod.tpl

```diff
@@ -73,11 +80,11 @@ containers:
       - name: vector
         containerPort: 6000
         protocol: TCP
-      - name: prometheus
+      - name: prom-exporter
```

Comment:
Here `prometheus` is being renamed to `prom-exporter`, but below `prometheus-exporter` is being renamed to `prom-exporter`. Is it ok that things that had different names now have the same name?

Reply:
So I found out that `containerPort` names can't be more than 15(16?) characters, which broke when autogenerating them based off `customConfig` - I wanted to consistently name things across `Pods` and `Services` (as well as the provided defaults and what the generated output is.

tl;dr I wanted to ensure that if you passed what I've provided as default into the `customConfig` option you'd get the same output as just using the defaults.